src/odemis/driver/smaract.py

```diff
@@ -3084,3 +3085,1139 @@ def SA_CTL_Move(self, handle, ch, pos_pm, _):
+            The device can also be addressed by its serial number:
+                usb:sn:<serial_number>
+                network:sn:<serial_number>
+        channels: (str --> int) dictionary mapping channel names to channel numbers
+        ref_on_init: (bool) determines if the controller should automatically reference
+            on initialization
+        precision_mode: (0 <= int <= 5) strength of digital lowpass filter, a higher level corresponds to higher
+            precision, but lower velocity. Not available on all systems.
+        """
+        model.HwComponent.__init__(self, name, role, *args, **kwargs)
+
+        if not channels:
+            raise ValueError("Needs at least 1 axis.")
+        self.channels = channels  # channel name -> channel number used by controller
+
+        # Connection
+        if locator == "fake":
+            self.core = FakePicoscale_DLL()
+        else:
+            self.core = SA_SIDLL()
+        self._locator = locator
+        self._id = self._openConnection(locator)
+
+        # Device information
+        devname = self.GetProperty_s(SA_SIDLL.SA_SI_DEVICE_NAME_PROP)
+        sn = self.GetProperty_s(SA_SIDLL.SA_SI_DEVICE_SERIAL_NUMBER_PROP)
+        num_ch = self.GetNumberOfChannels()
+        self._hwVersion = "SmarAct %s (s/n %s) with %s channels." % (devname, sn, num_ch,)
+        self._swVersion = self.GetFullVersionString()
+        logging.debug("Using Picoscale library version %s to connect to %s. ", self._swVersion, self._hwVersion)
+
+        # Device setup
+        self.EnableFullAccess()
+        try:
+            self.LoadConfiguration()
+        except SA_SIError as ex:
+            raise ValueError("Failed to load configuration. This might indicate that the device is not "
+                             "configured properly. Error: %s." % ex)
+
+        # Log configuration
+        flen = self.GetFiberLength()
+        ext_flen = self.GetExtensionFiberLength()
+        wdist_min, wdist_max = self.GetWorkingDistance()
+        logging.debug("Picoscale configuration: fiber length %s, extension fiber length %s, "
+                      "working distance [%s, %s].", flen, ext_flen, wdist_min, wdist_max)
+
+        # Position polling
+        self.position = model.VigilantAttribute({}, getter=self._getPosition, readonly=True)
+        self._polling_thread = util.RepeatingTimer(1, self._updatePosition, "Position polling")
+        self._polling_thread.start()
+
+        # Precision mode
+        # The precision mode is a special feature that needs to be purchased separately, so it is not available
+        # by default.
+        # TODO: this functionality has not yet been tested
+        if precision_mode != 0:
+            if not self.IsFeatureEnabled(SA_SIDLL.FEATURE_PRECISION_MODE):
+                raise ValueError("Device does not have precision mode. This feature needs to be purchased separately.")
+            self.SetPrecisionMode(precision_mode)
```

Comment:
You probably should set it to 0, if it's requested to be 0. Otherwise it won't be possible to disable it,